packages/node/src/integrations/http.ts

```diff
@@ -96,22 +109,23 @@ function _createWrappedHandlerMaker(breadcrumbsEnabled: boolean, tracingEnabled:
         transaction = scope.getTransaction();
         if (transaction) {
           span = transaction.startChild({
-            description: `${typeof options === 'string' || !options.method ? 'GET' : options.method} ${requestUrl}`,
+            description: `${requestOptions.method || 'GET'} ${requestUrl}`,
             op: 'request',
           });
+          addSentryTraceHeader(span, requestOptions);
         }
       }
 
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      return originalHandler
-        .apply(this, arguments)
+      return originalRequestMethod
+        .apply(this, requestArgs)
         .once('response', function(this: http.IncomingMessage, res: http.ServerResponse): void {
           if (breadcrumbsEnabled) {
             addRequestBreadcrumb('response', requestUrl, this, res);
           }
           if (tracingEnabled && span) {
             span.setHttpStatus(res.statusCode);
-            cleanDescription(options, this, span);
+            cleanSpanDescription(requestOptions, this, span);
```

Comment:
I'm not sure if it's a widely used convention, but whenever I write methods like that, I prefer the order of arguments to always be: `(subject, [...modifiers])`, so in this case, it'd be ` (span, module, methodOptions` (or `requestOptions`). I'd also do `const module = this` as we are at it, to actually indicate what `this` is

Reply:
Done.